apiserver/facades/agent/machine/machiner.go

```diff
@@ -5,7 +5,6 @@ package machine
 
 import (
 	"context"
-
 	"github.com/juju/clock"
 	"github.com/juju/errors"
 	"github.com/juju/names/v6"
```

Comment:
This now doesn't match the styleguide

https://github.com/juju/juju/blob/main/STYLE.md#imports